.travis.yml

```diff
@@ -1,8 +1,24 @@
 language: python
 python:
-    - "3.5"
+  - 2.7
+  - 3.4
+  - 3.5
+env:
+  - DJANGO=18
+  - DJANGO=19
+  - DJANGO=110
+  - DJANGO=111
+  - DJANGO=master
 sudo: false
 install:
   - pip install tox
 script:
-  - tox
+  - tox -e py${TRAVIS_PYTHON_VERSION//./}-dj${DJANGO}-lpclient4
```

Comment:
`lpclient4` and `lpclient5` should probably be some environment variable as well. I haven't figured out how to set them to build combine them with ever `DJANGO` version.